app/models/category.rb

```diff
@@ -124,8 +141,12 @@ def gid
     self.class.gid(id)
   end
 
-  def full_name
-    ancestors.reverse.map(&:name).push(name).join(" > ")
+  def name(locale: "en")
+    self.class.find_localization(locale, id, "name") || super()
+  end
+
+  def full_name(locale: "en")
+    ancestors.map { _1.name(locale:) }.push(name(locale:)).join(" > ")
```

Comment:
Lost the `reverse` here
```suggestion
    ancestors.reverse.map { _1.name(locale:) }.push(name(locale:)).join(" > ")
```